resources/warnings.json

```
@@ -13282,7 +13282,7 @@
     "versions": [
       {
         "lastVersion": "1.15",
-        "pattern": ".*"
+        "pattern": "(0|1[.][0-9]|0|1[.][0-9][.][0-9]+|1[.]1[0-5])"
```

This regex includes the non-existing version `0` twice. It's not _wrong_, just unnecessary. Additionally, this expression is quite repetitive. Instead of using an optional part to cover both 1.x and 1.x.y, it has two separate branches for these.

The regex in the usual format of prefixes allowing any suffix not producing false positives would be the shorter:

```suggestion
        "pattern": "(1[.][1-9]|1[.]1[0-5])(|[.-].+)"
```